default/techs.txt

```diff
@@ -2276,7 +2291,7 @@ Tech
     researchturns = 8
     prerequisites = [
         "SHP_MULTICELL_CAST"
-        "GRO_LIFECYCLE_MAN"
+        "SHP_MONSTER_MATURATION"
```

Comment:
That's my take really, I don't like the "long line" argument at all, the length of the line depends entirely on how you chose to display the tech tree and has nothing at all to do with game balance, whereas adding interstitial techs slows to an already slow-to-research hull line isn't necessary.

The organic line's already got so many long drawn out waits between different parts being available, playing them recently I think the Endomorphic probably needs to be quicker to unlock not slower.

Also, if you're playing low or no monsters this tech becomes completely pointless, not keen on making optional extra style flavour into an already theoretical prerequisite overloaded tech tree.